pkg/bpf/bpf.go

```diff
@@ -204,21 +205,38 @@ func (l *BpfLoader) Stop() {
 
 func NewVersionMap(config *options.BpfConfig) *ebpf.Map {
 	var versionPath string
+	var versionMap *ebpf.Map
 	if config.AdsEnabled() {
-		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh/map/")
+		versionPath = filepath.Join(config.BpfFsPath + constants.VersionPath)
```

Comment:
We can up[date later
```suggestion
		versionPath = filepath.Join(config.BpfFsPath, constants.VersionPath)
```